style/style.css

```diff
@@ -632,3 +727,449 @@ form .line {
   left: 120px;
   margin-bottom: 8px;
 }
+
+@media (min-width: 768px) {
+  body {
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  nav {
+    margin-top: -15px;
+  }
+
+  ul {
+    display: flex;
+    align-items: center;
+    float: right;
+    gap: 40px;
+    margin-right: 143px;
+    margin-top: -47px;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  li a {
+    width: 63px;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    margin-top: -15px;
+  }
+
+  .info {
+    background: url("../style/img/desktop/Header\ bg@2x.png");
+    background-size: cover;
+    width: 1440px;
+    margin-bottom: 180px;
+    position: relative;
+    left: 500px;
+    top: 100px;
+  }
+
+  .connect1 {
+    top: 15px;
+  }
+
+  .connect1 .text1 {
+    position: relative;
+    left: -65%;
+  }
+
+  .connect2 .text2 {
+    margin-left: -180px;
+  }
+
+  .text1 a {
+    position: relative;
+    top: -7px;
+    color: hsl(234, 100%, 75%);
+    margin-left: 319px;
+  }
+
+  .logo .myLogo a {
+    margin-top: -47px;
+    margin-left: 192px;
+  }
+
+  .title1 {
+    width: 920px;
+    height: 122px;
+    margin-left: 120px;
+    padding-top: 170px;
+  }
+
+  .info .title1,
+  .presentation1,
+  .connect1,
+  .social-icons1 {
+    margin-left: -15%;
+  }
+
+  .title1 h1 {
+    width: 500px;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .presentation1 {
+    width: 1440px;
+    height: 916px;
+  }
+
+  .presentation1 .intro1 {
+    margin-top: 136px;
+    width: 920px;
+    height: 98px;
+    margin-left: 15px;
+    font-size: 20px;
+  }
+
+  .social-icons1 {
+    height: 20px;
+    display: flex;
+    margin: 5.5rem 0;
+    align-items: center;
+    width: 11rem;
+    gap: 10px;
+    position: relative;
+    top: -124px;
+    list-style: none;
+  }
+
+  .connect1 .social-icons1 {
+    position: relative;
+    left: -59%;
+  }
+
+  .allcards {
+    width: 100%;
+    position: relative;
+    background-color: hsl(0, 0%, 90%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-left: -40px;
+    padding: 142px 0;
+  }
+
+  .snapshoot1 {
+    display: none;
+  }
+
+  .snapshoot2 {
+    margin-top: -68px;
+    width: 50%;
+    padding: 10px;
+    display: flex;
+    height: 100%;
+  }
+
+  .left-block {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: 100%;
+    left: 0;
+  }
+
+  .card1,
+  .card3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 46px;
+    width: 70%;
+  }
+
+  .card2,
+  .card4 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+    width: 70%;
+  }
+
```

Comment:
- I think your cards may have been squashed because of the `width: 70%;` added to each of them. 
-  Here is what I mean 👇🏽 
![tight](https://user-images.githubusercontent.com/99246359/200010832-6b983703-60ac-4e91-bb1d-6f33e48ea976.PNG)

I think you can fix this by simply increasing the width to either 85% or 90%. Or what do you think?? 🤔